src/6_branch.js

```diff
@@ -462,6 +582,15 @@ Branch.prototype['init'] = wrap(
 									journeyInTestMode
 								);
 							}
+							// TODO: check for Twitter referrer, try to use a decision from backend to make this change dynamic without web sdk
+							else if (!branchMatchIdFromOptions && utils.getParamValue('branchify_url') && self._referringLink()) {
```

Comment:
Can you explain this logic?
- !branchMatchIdFromOptions: Just curious why?
- self._referringLink: Won't this not be referred?

Reply:
!branchMatchIdFromOptions: The thinking was if we have the branch_match_id, rely on it for matching.

self._referringLink: It will be set to the `branchify_url` after v1/open sets the link data on session storage. It's not really the same a `document.referrer`.

Reply:
Understood.